### PR TITLE
Adds a Desert Eagle to the traitor uplink.

### PR DESCRIPTION
**Pull request by dessysalta**
## About The Pull Request

Adds a Deagle (and its corresponding magazine) to the traitor uplink.
## Why It's Good For The Game

Mostly cosmetic difference to the .357, I figured more variety in firearms would be nice. Gives you the choice between being a traitor with a revolver to an internal affairs agent with a Deagle--etc etc.
## Changelog
:cl: dessysalta and dopamiin
add: Added deagle and .50 AE magazine to traitor uplink.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items/dangerous.dm [hunk 1]
```diff
@@ -93,6 +93,15 @@
...
 	restricted = TRUE
 
+/datum/uplink_item/dangerous/deagle
+	name = "Desert Eagle"
+	desc = "The tried-and-true, bulky behemoth of a pistol. Fires .50 AE rounds and contains 8 shots."
+	item = /obj/item/gun/ballistic/automatic/pistol/deagle
+	progression_minimum = 30 MINUTES
+	cost = 13
+	surplus = 50
+	purchasable_from = ~UPLINK_CLOWN_OPS
```
Comment (Fikou) [inline code/modules/uplink/uplink_items/dangerous.dm:103]: It cannot be nukie accessible.

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items/dangerous.dm (hunk 1 above)
Comment (dessysalta) [reply code/modules/uplink/uplink_items/dangerous.dm]: Done, I think!

**Review by Fikou — APPROVED** [review]
dont know if we need this kind of variety but im fine with it

**Comment by Livrah** [comment]
Deagle is much easier to reload (.357 speedloader cost 3 extra TC)

**Comment by Sealed101** [comment]
doesnt deagle have good melee damage as well or something

**Comment by dessysalta** [comment]
Would dropping the mag price to 2/gun itself to 12 make it better? I considered making it 12 on the initial pull, but Dopa said it'd be a good idea to leave it as 13 if I wanted it to get merged a bit quicker.

File: code/modules/uplink/uplink_items/ammunition.dm [hunk 2]
```diff
@@ -59,6 +59,16 @@
 	cost = 2
 	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
 
+/datum/uplink_item/ammo/deagle
+	name = ".50 AE Magazine"
+	desc = "An additional 7-round .50 AE magazine, compatible with the Desert Eagle pistol. \
+			Just in case you need to turn your enemies OFF."
```
Comment (Mothblocks) [inline code/modules/uplink/uplink_items/ammunition.dm:65]:
```suggestion
		Just in case you need to turn your enemies OFF."
```

no tab alignment

Comment (dessysalta) [reply code/modules/uplink/uplink_items/ammunition.dm]: Done, or should be!

Comment (dessysalta) [reply code/modules/uplink/uplink_items/ammunition.dm]: Oh whoops, I pulled it back too many tab aligns. Just a second.

Comment (dessysalta) [reply code/modules/uplink/uplink_items/ammunition.dm]: Wait, the .357 speedloader has the same amount of indention the .50 AE mag does. Should I keep it as is (how it is now) or re-add an indent?

Comment (Mothblocks) [reply code/modules/uplink/uplink_items/ammunition.dm]: Up to you, we don't require changing old code but it's correct to fix it

Comment (dessysalta) [reply code/modules/uplink/uplink_items/ammunition.dm]: I'll knock all the unneeded indents out and then resolve this when I have. Thanks!

Comment (dessysalta) [reply code/modules/uplink/uplink_items/ammunition.dm]: Should be done! Let me know if it isn't.

File: code/modules/uplink/uplink_items/ammunition.dm [hunk 3]
```diff
@@ -35,7 +35,7 @@
 /datum/uplink_item/ammo/pistolap
 	name = "9mm Armour Piercing Magazine"
 	desc = "An additional 8-round 9mm magazine, compatible with the Makarov pistol. \
-			These rounds are less effective at injuring the target but penetrate protective gear."
+	These rounds are less effective at injuring the target but penetrate protective gear."
```
Comment (Mothblocks) [inline code/modules/uplink/uplink_items/ammunition.dm:38]: These need to still be indented up ONE, just not two. They shouldn't align, just indicate something new.

Comment (dessysalta) [reply code/modules/uplink/uplink_items/ammunition.dm]: Should be done now.

**Comment by Valorium-RB** [comment]
It's been a while since we got some new ranged Syndie gear – this seems pretty fun! Plus, the higher melee damage is balanced out by the much lower ammo availability, IMO. Deagles are stylish and novel. +1.

**Comment by optimumtact** [comment]
Things the uplink doesn't need

more guns

**Comment by dessysalta** [comment]
> Things the uplink doesn't need
> 
> more guns

Can I get an elaboration on this, more specifically on why a somewhat cosmetic change that adds an underused weapon into the uplink is a bad decision? To get less than a sentence of text after making an effort to include something like this is really disheartening.

**Comment by optimumtact** [comment]
>Can I get an elaboration on this, 

Uplink - the traitor uplink with traitor items

Guns - a projectile weapon that shoots energy or physical projectiles at a target

**Comment by optimumtact** [comment]
I don't know how to be any clearer

**Comment by dessysalta** [comment]
> >Can I get an elaboration on this, 
> 
> Uplink - the traitor uplink with traitor items
> 
> Guns - a projectile weapon that shoots energy or physical projectiles at a target

This doesn't clarify any of my views nor answer my question; it's sarcastic and honestly upsetting. Why are guns in their entirety (flavor, new, etc) _not_ a good PR decision, _specifically_ in this case?

I feel as though the flavor addition of a deagle (an underused and honestly somewhat unknown weapon to the station) to create more thorough gimmicks (internal affairs agents would be more likely to pick a deagle, maybe some kind of DMC reference?) is an entirely different addition than the intention of "the uplink doesn't need more guns". In the end, this PR doesn't practically change much, but it's that little difference that I think adds to the experience. 

Generally, I just don't understand how the intention of "the uplink doesn't need more guns" (I'm assuming that to be things like complicated or blatantly unneeded weapons) meets with my PR. I picked it specifically because of moments on Manuel I thought a deagle would play better with my gimmick or outfit. If there's concern of bloating the uplink, I understand, but no guns _outright?_ Am I missing anything?

This is all coming from a good place, I promise. It just felt really dismissive to get multiple people (an admin included) to support it only to be stonewalled and get less than a sentence of explanation.

**Comment by Serxule** [comment]
oranges after not allowing fun (he doesnt need it in "his" server)

**Comment by Mothblocks** [comment]
its just a gun theres already guns serxule

**Pull request closed without merging**